cltk/lemmatize/latin/backoff.py

```diff
@@ -518,55 +330,291 @@ def _randomize_data(train, seed):
-        #lemmatizer = backoff7
-        self.lemmatizer = self.backoff6
+        self.backoff1 = IdentityLemmatizer(VERBOSE=self.VERBOSE)
+        self.backoff2 = DictLemmatizer(lemmas=self.LATIN_OLD_MODEL, source='Morpheus Lemmas', backoff=self.backoff1, VERBOSE=self.VERBOSE)
+        self.backoff3 = RegexpLemmatizer(self.latin_sub_patterns, source='CLTK Latin Regex Patterns', backoff=self.backoff2, VERBOSE=self.VERBOSE)
+        self.backoff4 = UnigramLemmatizer(self.train_sents, source='CLTK Sentence Training Data', backoff=self.backoff3, VERBOSE=self.VERBOSE)
+        self.backoff5 = DictLemmatizer(lemmas=self.LATIN_MODEL, source='Latin Model', backoff=self.backoff4, VERBOSE=self.VERBOSE)
+        self.lemmatizer = self.backoff5
 
     def lemmatize(self, tokens):
-        #lemmatizer = self._define_lemmatizer()
         lemmas = self.lemmatizer.lemmatize(tokens)
         return lemmas
 
     def evaluate(self):
-        #lemmatizer = self._define_lemmatizer()
-        return lemmatizer.evaluate(self.test_sents)
 
+        if self.VERBOSE:
+            raise AssertionError("evaluate() method only works when VERBOSE=False")
+        return self.lemmatizer.evaluate(self.test_sents)
+
+    def __repr__(self):
+        return f'<BackoffLatinLemmatizer v0.2>'
+
+
+if __name__ == '__main__':
+
+    from pprint import pprint
+    # l1 = DefaultLemmatizer('UNK', VERBOSE=True)
+    # l2 = DictLemmatizer(lemmas={'arma': 'arma', 'uirum': 'uir'}, backoff=l1, VERBOSE=True)
+    # l3 = UnigramLemmatizer(train=[[('cano', 'cano'), ('.', 'punc')],], backoff=l2, VERBOSE=True)
+    # l4 = RegexpLemmatizer(regexps=[('(.)tat(is|i|em|e|es|um|ibus)$', r'\1tas'),], backoff=l3, VERBOSE=True)
+    # lemmas = l4.lemmatize('arma uirum -que cano nobilitatis .'.split())
+    # pprint(lemmas)
+
+# [('arma', 'arma', <UnigramLemmatizer: [[('res', 'res'), ...], ...]>),
+# ('uirum', 'uir', <UnigramLemmatizer: [[('res', 'res'), ...], ...]>),
+# ('-que', '-que', <DictLemmatizer: {'!': 'punc', ...}>),
+# ('cano', 'cano', <DictLemmatizer: {'-nam': 'nam', ...}>),
+# ('nobilitatis',
+# 'nobilitas',
+# <RegexpLemmatizer: [('(bil)(is|i|e...es|ium|ibus)$', '\\1is'), ...]>),
+# ('.', 'punc', <DictLemmatizer: {'!': 'punc', ...}>)]
+
+    print('\n')
+
+    bll = BackoffLatinLemmatizer(seed=5, VERBOSE=False)
+    lemmas = bll.lemmatize('arma uirum -que cano nobilitatis .'.split())
+    pprint(lemmas)
+
+# [('arma', 'arma', <UnigramLemmatizer: CLTK Sentence Training Data>),
+# ('uirum', 'uir', <UnigramLemmatizer: CLTK Sentence Training Data>),
+# ('-que', '-que', <DictLemmatizer: Latin Model>),
+# ('cano', 'cano', <DictLemmatizer: Morpheus Lemmas>),
+# ('nobilitatis', 'nobilitas', <RegexpLemmatizer: CLTK Latin Regex Patterns>),
+# ('.', 'punc', <DictLemmatizer: Latin Model>)]
+
+    print(f'Accuracy: {bll.evaluate()}')
 
-# Accuracty test available below——keep? delete?
-#if __name__ == "__main__":
+# Accuracy: 0.899449857493206
+
```

Comment:
I vote to delete commented out code:
accuracy tests are good but probably belong in a notebook in a separate folder or a separate repo.
Also, as long as they are somewhere in git with a commit hash, they are still available.
thoughts? @kylepjohnson

Reply:
This is fine—I left a lot of in for the review. Can just take it out now.